### ISSUE TITLE
Page missing a title

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#674]
[Page missing a title - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402220/Page.missing.a.title.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)

**Comment by pooriamehregan** [comment]
Falsk positiv for lenker som gjelder portal.
